ocaml/xapi/xapi_clustering.ml

```diff
@@ -401,3 +403,68 @@ let compute_corosync_max_host_failures ~__context =
     ((nhosts - disabled_hosts - 1) / 2) + disabled_hosts
   in
   corosync_ha_max_hosts
+
+let on_corosync_update ~__context ~cluster update =
```

Comment:
The error messages going into the logs could use `"%s: .." __FUNCTION__` to make it more obvious where they are coming from.